examples/spark-ml-app-demo/emr/archetype/liminal.yml

```diff
@@ -0,0 +1,67 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+# superliminal for local development
+name: InfraSpark
+type: super
+variables:
+  region: us-east-1
+  core_count: 2
+  emr_version: 'emr-6.2.0'
+  template_url: 'https://s3.amazonaws.com/myorg/emr-template.yml'
+  cluster_name: 'spark-ml-app-demo-liminal'
+  spark_output_path: 'myorg/'
+executors:
+  - executor: emr-executor
+    cluster_name: '{{cluster_name}}'
+    type: emr
+  - executor: k8s-executor
+    type: kubernetes
+task_defaults:
+  spark:
+    executor: emr-executor
+    master: yarn
+    deploy_mode: cluster
+    application_source: 'myorg/{{application}}'
```

Comment:
Shouldn't we have an s3 path here? We could do one with a configurable bucket name